src/rgw/rgw_admin.cc

```diff
@@ -3227,6 +3232,8 @@ int main(int argc, const char **argv)
 
   SimpleCmd cmd(all_cmds, cmd_aliases);
 
+  boost::optional<std::string> rgw_obj_fs; // radoslist field separator
```

Comment:
nit, why not using `std::optional<>`. see https://en.cppreference.com/w/cpp/utility/optional